site/docs/v1/tech/premium-features/scim/azure-ad.adoc

```diff
@@ -0,0 +1,34 @@
+---
+layout: doc
+title: Azure AD SCIM Client
+description: Configure Azure AD as a SCIM client
+navcategory: premium
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+This is a basic example to get started configuring Azure AD as a SCIM client and establish a connection between FusionAuth and Azure AD for provisioning purposes.
+
+For complete documentation, please refer to the Azure AD provisioning documentation.
```

Comment:
Is there a link we can offer to folks?

Reply:
Not really, their doc is a loop of endless links. I ended up using 3rd party doc to figure it out. I could can add a link to the root of their doc and let them search.

Reply:
🤯 😬